modules/sdk/gradle-plugins-poshi-runner/src/main/java/com/liferay/gradle/plugins/poshi/runner/PoshiRunnerPlugin.java

```diff
@@ -274,6 +285,25 @@ public void execute(Task task) {
 
 			});
 
+		PluginContainer pluginContainer = project.getPlugins();
+
+		pluginContainer.withId(
+			"com.liferay.test.integration",
+			new Action<Plugin>() {
+
+				@Override
+				public void execute(Plugin plugin) {
+
+					test.dependsOn("_START_TESTABLE_TOMCAT_TASK_NAME");
```

Comment:
You created the `_START_TESTABLE_TOMCAT_TASK_NAME` and `_STOP_TESTABLE_TOMCAT_TASK_NAME` constants, but you're wrapping their names with quotes here, it means you're using the name of the constants as the actual value.